redux_userlist_06/src/components/Dashboard/withLoading.tsx

```diff
@@ -0,0 +1,12 @@
+
+function withLoading(Component:any){
+
+    return (function withLoadingComponent(props:any){
+        if(!props.isLoading)
+            return (<Component isLoading = {props.isLoading}/>)
+        else {
+            return <h1>Wait...loading</h1>
+        }
+    })
```

Comment:
use conditonal returning like this 

return (
  {
    props.isLoading ? 
    (<h1>Wait...loading</h1>): 
    (<Component isLoading = {props.isLoading}/>)
  })